bootstrap/helpers/versions.php

```
@@ -0,0 +1,53 @@
+ */
+function get_versions_data(): ?array
+{
+    return Cache::remember('coolify:versions:all', 3600, function () {
+        $versionsPath = base_path('versions.json');
+
+        if (! File::exists($versionsPath)) {
+            return null;
+        }
+
+        return json_decode(File::get($versionsPath), true);
+    });
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\Cache' in method 'get_versions_data'. (undefined)

(StaticAccess)